cmd/distrogen/distribution.go

```diff
@@ -295,6 +331,15 @@ func getGeneratedFilesInDir(dir string) (map[string]*generatedFile, error) {
 			return nil
 		}
 
+		// Don't include .tools directory in comparison.
+		if strings.Contains(path, ".tools") {
```

Comment:
Another super nitpick.

Since we already exclude all directories in the previous if statement. I think checking for `/.tools/` in the path should still exclude contents in the `.tools` directory while excluding false positives like files/directories that contains the `.tools` substring like `google.otel.tools` (although it's super unlikely)

Reply:
Excluding all directories just excludes the individual entry for that directory; `filepath.WalkDir` will still recursively walk the subdirectory even if we ignore the specific directory entry it presented us. So the check is still necessary.

Applied suggestion to make the check stricter in [8dc4905](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/286/commits/8dc4905b87df04316c5577fd633d55b8ddad9e64)